packages/svelte-example/README.md

```diff
@@ -0,0 +1,5 @@
+# Mosaic React / Svelte Example
```

Comment:
I would prefer not to populate the "packages" folder with examples. (I've wanted to move or remove the Vega-Lite example for some time, but just haven't gotten around to it.)

Instead, I'd like to move examples to a separate mono-repo folder. For example, maybe we should create a top-level `examples` folder and put all example packages there instead.